src/main/java/Flash.java

```diff
@@ -0,0 +1,107 @@
+import java.sql.Array;
+import java.util.Scanner;
```

Comment:
well done. code functionality is implemented with well organized methods and with naming that describes the purpose well, and is in camelCase. if else statement is also in egyptian format